### PR TITLE
Fixed target edit form grid gaps

### PR DESCRIPTION
**Pull request by diego-ge**
Changed target edit form grid gap for better alignment among fields
image

### CODE REVIEW

File: common/src/main/webapp/less/style.less [hunk 1]
```diff
@@ -1206,7 +1215,7 @@ tfoot {
 
 .explore-units-label {
   margin-left: 0.25em;
-  margin-bottom: 0.2em;
+  // margin-bottom: 0.2em;
```
Comment (rpiaggio) [inline common/src/main/webapp/less/style.less:1218]: Commented code

**Review by rpiaggio — APPROVED** [review]
LGTM

File: common/src/main/webapp/less/style.less (hunk 1 above)
Comment (diego-ge) [reply common/src/main/webapp/less/style.less]: Sorry about that.
Already deleted

**Comment by cquiroz** [comment]
You need no rebase this

**Pull request merged**